cetmix_tower_server/security/ir.model.access.csv

```diff
@@ -50,3 +50,4 @@ access_server_template_manager,Server Template->Manager,model_cx_tower_server_te
 access_server_template_root,Server Template->Root,model_cx_tower_server_template,group_root,1,1,1,1
 access_create_server_from_template_manager,Create Server From Template->Manager,model_cx_tower_server_template_create_wizard,group_manager,1,1,1,1
 access_create_server_from_template_line_manager,Create Server From Template Line->Manager,model_cx_tower_server_template_create_wizard_line,group_manager,1,1,1,1
+access_cx_tower_variable_option_user,cx_tower_variable_option_user,model_cx_tower_variable_option,base.group_user,1,1,1,1
```

Comment:
From the task: ` It must have the same access rules as the cx.tower.variable.value​ model.`

Reply:
Okay. I will fix it